src/transports/PostMessageTransport.ts

```diff
@@ -0,0 +1,90 @@
+import { Transport } from "./Transport";
+import { JSONRPCRequestData, IJSONRPCData } from "../Request";
+
+const openPopup = (url: string) => {
```

Comment:
mind exposing width height param with some reasonable default, ideally, left and top too. Then apps that are using this can real time size the popup height/width to their preference